src/platform/plugins/shared/chart_expressions/expression_heatmap/public/expression_renderers/heatmap_renderer.tsx

```diff
@@ -40,6 +42,15 @@ interface ExpressioHeatmapRendererDependencies {
   getStartDeps: StartServicesGetter<ExpressionHeatmapPluginStart>;
 }
 
+const heatmapContainerCss = ({ euiTheme }: UseEuiTheme) =>
+  css({
+    width: '100%',
+    height: '100%',
+    padding: euiTheme.size.s,
+    overflow: 'auto hidden',
```

Comment:
nit: It would be good not to loose a comment of why we add these styles
```suggestion
    // the FocusTrap is adding extra divs which are making the visualization redraw twice
    // with a visible glitch. This make the chart library resilient to this extra reflow
    overflow: 'auto hidden',
```